setup.py

```diff
@@ -203,7 +203,7 @@ def build_extensions(self):
 
 
 def get_package_version():
-    return __version__ + "+" + os.environ['HOROVOD_LOCAL_VERSION'] if 'HOROVOD_LOCAL_VERSION' in os.environ else __version__
+    return __version__ + "." + os.environ['HOROVOD_LOCAL_VERSION'] if 'HOROVOD_LOCAL_VERSION' in os.environ else __version__
```

Comment:
While touching this line, I would suggest to hardening it against `HOROVOD_LOCAL_VERSION=`
```suggestion
    return __version__ + "." + os.environ['HOROVOD_LOCAL_VERSION'] if os.environ.get('HOROVOD_LOCAL_VERSION') else __version__
```